### PR TITLE
chore: documentation for backups in platform

### PR DESCRIPTION
**Pull request by dragarcia**
* First pass of documentation for available forms of backups for hosted Supabase projects
* Screenshots to follow later on 
    * Is there a certain way of taking screenshots?

### CODE REVIEW

File: apps/reference/docs/guides/platform/backups.mdx [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+id: backups
+title: Backups
+description: Available forms of backups for your Supabase project
+---
+
+## Database Backups
```
Comment (dannykng) [inline apps/reference/docs/guides/platform/backups.mdx:7]: style nit: I don't think we need a heading immediately following the title
Screen Shot 2022-10-26 at 3 01 16 PM

```suggestion
---
id: database-backups
title: Database Backups
description: Learn about the different ways your Supabase project's database is backed up
---
```

File: apps/reference/docs/guides/platform/backups.mdx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+Database backups^ are an integral part of any disaster recovery plan. Disasters come in many shapes and sizes. It could be as simple as accidentally deleting a table column, the database crashing, or even a natural calamity wiping out the underlying hardware a database is running on. The risks and impact brought by these scenarios can never be fully eliminated, but only minimized or even mitigated. Having database backups is a form of insurance policy. They are essentially snapshots of the database at various points in time. When disaster strikes, database backups allow the project to be brought back to any of these points in time, therefore averting the crisis.
+
+### Frequency of Backups
```
Comment (dannykng) [inline apps/reference/docs/guides/platform/backups.mdx:11]:
```suggestion
## Frequency of Backups
```

File: apps/reference/docs/guides/platform/backups.mdx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+Insert screenshot of the Dashboard of the project in restoration mode
+```
+
+As the restoration occurs, the project on the Dashboard will be inaccessible as well. The Dashboard will display a notification once the restoration completes.
+
+```
+Insert screenshot of the Dashboard of the project completing restoration
+```
+
+---
```
Comment (dannykng) [inline apps/reference/docs/guides/platform/backups.mdx:56]: style nit: I don't think we need horizontal lines (kind of signifies it should be a separate doc/page)
```suggestion
```

**Review by dannykng — COMMENTED** [review]
Left some feedback but this looks great!

Regarding Dashboard screenshots: We don't have "official" style guidelines for that yet. Maybe for now just make sure they:
- are ~2000px wide or greater (is this too big?)
- are in jpg or png format
- don't contain any sensitive information

How does that sound?

**Comment by saltcod** [comment]
Hey @dragarcia  quick note that we've just migrated our docs from Docusaurus to NextJS. Apologies for the inconvenience, but would you mind porting these changes to `/apps/docs` instead of `docs/reference`?  You could wait until you have approval here and then move them over, or just do it now would work as well.

Any questions please let me know

Thanks and apologies in advance!

**Pull request merged**